client/my-sites/earn/memberships/index.jsx

```diff
@@ -697,10 +705,25 @@ class MembershipsSection extends Component {
 	}
 }
 
+/**
+ * Source is used to add data to the Stripe Connect URL. On a successful
+ * connection, this source is used to redirect the user the appropriate place.
+ */
+const getSource = () => {
+	if ( window.location.hash === ADD_NEWSLETTER_PAYMENT_PLAN_HASH ) {
+		return 'earn-newsletter';
+	}
+	if ( window.location.hash === LAUNCHPAD_HASH ) {
+		return 'launchpad';
```

Comment:
Since there will be many launchpads (one at onboarding, one at my home, one at earn), it would make sense to make this more specific. Let's use for example `setup-launchpad` to refer to the one in stepper?

Reply:
I've set this to full-launchpad. This follows the launchpad_screen option, which can be set to off, full, or minimized.